nikola/plugins/command/auto/__init__.py

```diff
@@ -429,7 +429,7 @@ async def reload_page(self, event) -> None:
         # Move events have a dest_path, some editors like gedit use a
         # move on larger save operations for write protection
         if event:
-            event_path = event.dest_path if hasattr(event, 'dest_path') else event.src_path
+            event_path = event.dest_path if (hasattr(event, 'dest_path') and event.dest_path) else event.src_path
```

Comment:
```suggestion
            event_path = getattr(event, 'dest_path', None) or event.src_path
```
This was suggested in #3766 and also works, not sure which is more Pythonic tho.

Reply:
I think the suggested change is less Pythonic.